### ISSUE TITLE
On reproducing the results of GIAS

### ISSUE THREAD

**Issue by honggudrnjs** [ml-postech/gradient-inversion-generative-image-prior#2]
First of all, thank you for sharing the code and dataset information related to GIAS.
I tried to reproduce inversion attack results using the script provided as a default on ImageNet dataset (Filename : ours_vw_bs4.sh).
However, the inverted images were somewhat different from the ground truth, but the images seem to be natural probably thanks to better image prior induced by GAN. Reconstruction loss during training was not that high, seems normal to me.
Should I customize hyperparameters in the script file for better result? I use ImageNet images though.
Could you give some suggestions for me? I tried FFHQ dataset also with the same script, but the inverted images were far from ground truth images.

**Comment by ffhibnese** [comment]
Hello, hong
I got the same issues as you when I tried to reproduce stylegan2 on FFHQ128. The genetared images are very unrealistic.
Could you please tell me have you solved this problem? I would be grateful if you can give me some advice.

**Comment by FrankMOWJ** [comment]
May I ask where can I download the checkpoint of pretrained StyleGan?

**Comment by ffhibnese** [comment]
> May I ask where can I download the checkpoint of pretrained StyleGan?

You can refer to my repository 'https://github.com/ffhibnese/GIFD_Gradient_Inversion_Attack', which presents detailed instructions for downloading pretrained model checkpoints.

Please give it a star if you find it useful :)
